src/Tooltip.js

```diff
@@ -9,12 +9,19 @@ const propTypes = {
   disabled: PropTypes.bool,
   tether: PropTypes.object,
   toggle: PropTypes.func,
-  children: PropTypes.node
+  children: PropTypes.node,
+  delay: PropTypes.oneOfType([PropTypes.object, PropTypes.number])
```

Comment:
Why not use PropTypes.shape to define the object instead of just the generic object since you are mandating that show and hide have to be a number in `getDelay`. It avoids the `NaN` check and would give developers a specific message when they don't use the right shape.